README.md

```diff
@@ -7,79 +7,79 @@ Command-line tool for deploying Fluo or Accumulo to a cluster that can be option
 Installation
 ------------
 
-First clone the `zetten` repo:
+First clone the `muchos` repo:
 
-    git clone https://github.com/fluo-io/zetten.git
+    git clone https://github.com/astralway/muchos.git
 
-Now, create and modify your configuration file for zetten:
+Now, create and modify your configuration file for muchos:
```

Comment:
Could capatilize this instance of Muchos.   In some cases we are refering to the command or repo name, but here it seems like its referring to the project name